### ISSUE TITLE
Consider adding an additional filtering option to only run analysis after Nth line hit

### ISSUE THREAD

**Issue by markerikson** [RecordReplay/devtools#6906]
Per https://twitter.com/bradleymeck/status/1528165954395447296 , it might be useful to have a breakpoint/print statement filtering option that only starts running analysis after the line has been hit N times.

**Comment by jaril** [comment]
My understanding of that tweet is that _shu would like to pause at the breakpoint right before a crash. For example, if a breakpoint was hit 1000 times but it crashes between 999 and 1000, then you'd like to pause at 999. In replay, that's super easy since you can just go to the end of the recording, and hit previous hit on that breakpoint twice and you're there. No need for any filtering.

I think that's also what you were trying to explain in [this tweet](https://twitter.com/acemarke/status/1528166621377974272), so this should already be resolved. Am I missing something? Let me know!

**Comment by markerikson** [comment]
Yeah, I'm suggesting that it _might_ still be useful to say "look, I know this line is gonna get hit 100 times, but skip logging the first 50 of them and only start printing output at 51", etc.

**Comment by jaril** [comment]
> Yeah, I'm suggesting that it _might_ still be useful to say "look, I know this line is gonna get hit 100 times, but skip logging the first 50 of them and only start printing output at 51", etc.

AH got it!

My sense is this is an edge case and a new filter implementation might conflict (and actually, is somewhat addressable) by focus mode. Which I'm just now realizing is exactly what you said [here](https://twitter.com/acemarke/status/1528166955504570370) 🤦‍♂️

Switching it to a discussion so @jonbell-lot23 @jcmorrow can weigh in.
